problemsets-mat223/modules/module9.tex

```diff
@@ -70,8 +70,28 @@
 		\mat{x\\y}\stackrel{T}{\mapsto}\matc{x\\y+4}.
 	\]
 	For each of $S$ and $T$, determine whether the transformation is linear.
+	
+	Let vectors $\vec u=\mat{u_1\\u_2}$, $\vec v=\mat{v_1\\v_2}$ and the scalar $\alpha$ be given.
 
-	XXX Finish
+	For $\mathcal S$:
+	
+	We need to verify that $\mathcal S(\vec u+\vec v)=\mathcal S(\vec u)+\mathcal S(\vec v)$ and $\mathcal S(a\vec u)=a\mathcal S(\vec u)$. Computing, we see:
+	\[
+	    \mathcal S(\vec u +\vec v)=\mathcal S\left(\mat{u_1+v_1\\u_2+v_2}\right)=\mat{2u_1+2v_1\\u_2+v_2}=\mat{2u_1\\u_2}+\mat{2v_1\\v_2}=\mathcal S(\vec u) + \mathcal S(\vec v)
+	\]
+	and
+	\[
+	    \mathcal S(a\vec u)=\mat{2a u_1\\a u_2}=\mathcal S\mat{a u_1\\a u_2}=a\mat{2 u_1\\ u_2}=a\mathcal S(\vec u).
+	\]
+	Therefore, $\mathcal S$ is linear.
+	
+	For $\mathcal T$:
+	
+	Notice that $\mathcal T(\vec u+\vec v)=\matc{u_1+v_1\\u_2+v_2+4}$ doesn't look like $\mathcal T(\vec u)+\mathcal T(\vec v)=\matc{u_1+v_1\\u_2+v_2+8}$. Therefore, we will guess that $T$ is not linear and look for a counter example. Using $\xhat=\mat{1\\0}$ and $\yhat=\mat{0\\1}$, we see:
```

Comment:
`we see:` -> `we see`

Break this into a couple of lines instead of one really long one.